sqlmesh/core/model/meta.py

```diff
@@ -277,7 +278,14 @@ def _kind_validator(cls, values: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
 
             if hasattr(kind, "unique_key"):
                 kind.unique_key = [
-                    normalize_identifiers(key, dialect=dialect) for key in kind.unique_key
+                    quote_identifiers(normalize_identifiers(key, dialect=dialect), dialect=dialect)
+                    for key in kind.unique_key
+                ]
+
+            if hasattr(kind, "columns") and isinstance(kind.columns, list):
```

Comment:
@eakmanrq I'm assuming this should've been added in order to support the `SCDType2ByColumnKind` kind correctly but was overlooked. Please confirm.

Reply:
what about time_column?

Reply:
should we make a function normalize_and_quote?

Reply:
`time_column` is a string, so the quoting seems meaningless.

Reply:
See: https://github.com/TobikoData/sqlmesh/blob/98a4610f575a06fb32bbcad5ca3b11e81d34e66b/sqlmesh/core/model/meta.py#L277

Reply:
why the heck is it a string

Reply:
it's been a long time lol

Reply:
Yes I agree thanks for catching this.